controller/provisioner.go

```diff
@@ -45,11 +45,23 @@ func (p *Provisioner) Provision(opts pvController.VolumeOptions) (*v1.Persistent
 		return nil, fmt.Errorf("ReadWriteMany access mode is not supported")
 	}
 	resourceStorage := pvc.Spec.Resources.Requests[v1.ResourceName(v1.ResourceStorage)]
-	numberOfReplicas, err := strconv.Atoi(opts.Parameters[types.OptionNumberOfReplica])
+	var numberOfReplicaParam, optionStaleReplicaTimeoutParam string
+	if _, ok := opts.Parameters[types.OptionNumberOfReplica]; !ok {
```

Comment:
In fact, let's do this instead: (notice it's numberOfReplicas, with `s`)
```
numberOfReplicasParam := types.DefaultNumberOfReplicas
if _, ok = opts.Parameters[types.OptionNumberOfReplicas]; ok {
    numberOfReplicasParam = opts.Parameters[types.OptionNumberOfReplica]
}
```

Same below.